### PR TITLE
Add vagrant

### PR DESCRIPTION
**Pull request by nyameteor**

### CODE REVIEW

File: data/vagrant [hunk 1]
```diff
@@ -0,0 +1,2 @@
+vagrantup.com
+vagrantcloud.com
```
Comment (IceCodeNew) [inline data/vagrant:2]: 请按照字典序排列

**Review by IceCodeNew — CHANGES_REQUESTED** [review]
.

**Comment by Loyalsoldier** [comment]
`vagrant` belongs to `hashicorp`, please update it in `hashicorp` instead.

File: data/vagrant (hunk 1 above)
Comment (nyameteor) [reply data/vagrant:2]: 好的，不好意思

**Comment by nyameteor** [comment]
> `vagrant` belongs to `hashicorp`, please update it in `hashicorp` instead.

Oh.. I didn't see `vagrantup.com` in `hashicorp`, my apologies.

I saw `vagrantcloud.com` domain from vagrant log, but when I try `curl --head -L https://vagrantcloud.com`, it just redirects to https://app.vagrantup.com/, does this domain need to be added?

**Comment by Loyalsoldier** [comment]
> > `vagrant` belongs to `hashicorp`, please update it in `hashicorp` instead.
> 
> Oh.. I didn't see `vagrantup.com` in `hashicorp`, my apologies.
> 
> I saw `vagrantcloud.com` domain from vagrant log, but when I try `curl --head -L https://vagrantcloud.com`, it just redirects to https://app.vagrantup.com/, does this domain need to be added?

Please add it.

**Comment by nyameteor** [comment]
> > > `vagrant` belongs to `hashicorp`, please update it in `hashicorp` instead.
> > 
> > 
> > Oh.. I didn't see `vagrantup.com` in `hashicorp`, my apologies.
> > I saw `vagrantcloud.com` domain from vagrant log, but when I try `curl --head -L https://vagrantcloud.com`, it just redirects to https://app.vagrantup.com/, does this domain need to be added?
> 
> Please add it.

OK, I have added it to `hashicorp`, thanks.

**Review by Loyalsoldier — APPROVED** [review]

**Pull request merged**